README.md

```diff
@@ -26,6 +34,12 @@
   width: 10px;
   height: 10px;
 }
+.minmax {
```

Comment:
```suggestion

.minmax {
```

Reply:
There’s an opportunity to improve consistency in the doc:

<img width="229" alt="image" src="https://github.com/postcss/postcss-size/assets/9340937/80b836db-5735-40ad-acf3-8916ed0dfd85">

Do you prefer an additional break between `.one` and `.two`, too? I’ll align input/output after your answer.

Reply:
Let's go without blank line